pkg/awx/connection.go

```diff
@@ -249,28 +250,26 @@ func (c *Connection) get(path string, query url.Values, output interface{}) erro
 func (c *Connection) rawGet(path string, query url.Values) (output []byte, err error) {
 	// Send the request:
 	address := c.makeUrl(path, query)
-	if glog.V(2) {
-		glog.Infof("Sending GET request to '%s'.", address)
-	}
 	request, err := http.NewRequest(http.MethodGet, address, nil)
 	if err != nil {
 		return
 	}
 	c.setAgent(request)
 	c.setCredentials(request)
 	c.setAccept(request)
+	if glog.V(2) {
+		glog.Infof("Sending GET request to '%s'.", address)
```

Comment:
This ^ line is now redundant, as it will be part of the result of `DumpRequest`.